node/node.go

```diff
@@ -222,7 +222,7 @@ func makeNode(cfg *config.Config,
 	}
 
 	// Determine whether we should attempt state sync.
-	stateSync := cfg.StateSync.Enable && !onlyValidatorIsUs(state, pubKey)
+	stateSync := !onlyValidatorIsUs(state, pubKey)
```

Comment:
Not sure I get this change. Is this rendering state sync mandatory?

Reply:
State sync cannot be mandatory.

Reply:
Yeah this doesn't make sense

Reply:
Sorry. Meant to make this change to the blocksync line which looks basically the same. Fixed!